instance/migrations/0124_openedxinstance_static_templates_overrides.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.10 on 2020-03-19 19:08
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+import instance.schemas.static_templates_overrides
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0123_newrelicsslalertcondition_newrelicsslmonitor'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='openedxinstance',
+            name='static_templates_overrides',
+            field=django.contrib.postgres.fields.jsonb.JSONField(blank=True, help_text="The final static templates' override data committed by the user for deployment. This should bepicked up the appservers launched for this instance.", null=True, validators=[instance.schemas.static_templates_overrides.static_templates_overrides_schema_validate], verbose_name='Draft static templates overrides JSON'),
```

Comment:
Nit: Missing space on `This should bepicked up the appservers` ("bepicked").

Reply:
@giovannicimolin, the source of this issue is in the field definition. So I will fix it here and there.